README.rst

```diff
@@ -33,28 +33,34 @@ Get started with tutorials
 
 * (Easiest) Improve a simple classifier from 60% to 80% accuracy on the Iris dataset:
   
-  * `TUTORIAL: simple cleanlab on Iris <https://github.com/cleanlab/examples/blob/master/iris_simple_example.ipynb>`__.
+  * `TUTORIAL: simple cleanlab on Iris <https://github.com/cleanlab/examples/blob/master/iris_simple_example.ipynb>`__
+
+* (Comprehensive) Image classification with noisy labels
+
+  * `TUTORIAL: learning with noisy labels on CIFAR <https://github.com/cleanlab/examples/tree/master/cifar10>`__
 
 * Run Cleanlab on 4 datasets using 9 different classifiers/models:
   
-  * `TUTORIAL: classifier comparison <https://github.com/cleanlab/examples/blob/master/classifier_comparison.ipynb>`__.
+  * `TUTORIAL: classifier comparison <https://github.com/cleanlab/examples/blob/master/classifier_comparison.ipynb>`__
 
-* Find `label errors <https://openreview.net/forum?id=XccDXrDNLek>`_ in MNIST, ImageNet, CIFAR-10/100, Caltech-256, QuickDraw, Amazon Reviews, IMDB, 20 Newsgroups, & AudioSet:
+* Find `label errors <https://arxiv.org/abs/2103.14749>`_ in MNIST, ImageNet, CIFAR-10/100, Caltech-256, QuickDraw, Amazon Reviews, IMDB, 20 Newsgroups, AudioSet:
```

Comment:
I personally prefer reading arxiv versions since they contain appendix. The neurips version is indeed linked in the new readme under "News! (2021)" pulldown, but I linked the arxiv version most other places. Can change it if you want

Reply:
arXiv is fine (double checked and the neurips publication is mentioned as a footnote on the arxiv, that should suffice and i agree arXiv is better to avoid broken links main --> supplemental)